x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_list.tsx

```diff
@@ -62,20 +71,22 @@ interface Props {
   isManagementTable?: boolean;
   isMlEnabledInSpace?: boolean;
   blockRefresh?: boolean;
-  openCreateJobModal?: ActionDispatchers['openModal'];
+  createAnalyticsForm?: any;
```

Comment:
You could import `CreateAnalyticsFormProps` from `use_create_analytics_form.ts` and provide typing somewhat like `createAnalyticsForm?: () => CreateAnalyticsFormProps`.

Reply:
Resolved in https://github.com/elastic/kibana/pull/49464/commits/c4603cd3451f1ac309bddca0b8150499c7fffffa